CHANGELOG.md

```diff
@@ -7,16 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
-- [#1](https://github.com/qbic-pipelines/variantmtb/pull/1) - Query to CGI & CIViC. Creation of a comprehensive HTML report. 
-
+- [#1](https://github.com/qbic-pipelines/variantmtb/pull/1) - Query to CGI & CIViC. Creation of a comprehensive HTML report.
```

Comment:
did you change anything here?
I think we dont have to but only create a changelog when we push to master. However, maybe we should save everything we change somewhere

Reply:
seems like my prettier run did this

Reply:
ah ok. sure looks like it